src/lib/shared/models/props.model.ts

```diff
@@ -23,9 +23,9 @@ export interface FaProps {
   counter?: boolean;
   inverse?: boolean;
   flip?: FlipProp;
-  size?: SizeProp;
-  pull?: PullProp;
-  rotate?: RotateProp;
+  size?: SizeProp | null;
```

Comment:
Can you explain why this change is needed?

Reply:
turning off strictNullChecks fixes it, reverted